analysis/plotScores.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import csv
+import matplotlib.pyplot as plt
+
```

Comment:
@lpanfili are you familiar with [the PEP8 code style guidelines](https://www.python.org/dev/peps/pep-0008/)?  It may be valuable in future to be familiar with them (or better yet, to have an editor that automatically checks your code against PEP8 and tells you which lines deviate and how to fix them.  At first glance, I notice that there should be [2 blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines) before new to-level `def` statements.